lib/main_common.pm

```diff
@@ -1654,8 +1654,19 @@ sub load_wicked_tests {
         loadtest 'wicked/config_files';
     }
     elsif (check_var('WICKED', 'advanced')) {
-        loadtest 'wicked/advanced_ref' if check_var('IS_WICKED_REF', '1');
-        loadtest 'wicked/advanced_sut' if check_var('IS_WICKED_REF', '0');
+        my $f = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
```

Comment:
sorry I was a bit late with my change.
Plz use `check_var('IS_WICKED_REF', '1');` I just saw that IS_WICKED_REF is set for both

Reply:
Ah no it will work in both ways.

Reply:
as far as I know perl will treat '0' as false , no ?